versions/1/_dev/build/spec.yml

```diff
@@ -0,0 +1,8 @@
+spec:
+  additionalContents: false
+  contents:
+    - description: Folder containing resources related to docs.
+      type: folder
+      name: docs
+      required: false
```

Comment:
Should we make this `true`? IOW, should every package be **required** to have a README?

Reply:
No. It's possible to have a package without a README template. The one that doesn't have at the moment in the `log` package..

Reply:
Okay, but does it make sense for the `log` package to have a README? Or to ask differently: what would be the downside of defining a README for the `log` package?

Reply:
You mean, a README file template.

We can it there, in this case both files will have the same content.

Reply:
Yes, this is partly related to my confusion as described in https://github.com/elastic/package-spec/pull/84#issuecomment-729541408 😄 . 

Quick questions:
- say a developer defines a README template for a package but also accidentally edits the README by hand such that it's no longer in sync with the template. I assume our tooling would catch this, maybe as part of `elastic-package lint` or something?
- for packages that define a README template, it sounds like a developer would run `elastic-package build` to (re)generate the README. Normally, `elastic-package build` places the resulting package into the `build` folder, but in this case it sounds like it would also place an updated `README.md` in the package source folder itself, right?

Reply:
> say a developer defines a README template for a package but also accidentally edits the README by hand such that it's no longer in sync with the template. I assume our tooling would catch this, maybe as part of elastic-package lint or something?

Yes, the `lint` command can detect it in advance. Keep in mind that when developer runs `build` command, it will overwrite the README.md.

>for packages that define a README template, it sounds like a developer would run elastic-package build to (re)generate the README. Normally, elastic-package build places the resulting package into the build folder, but in this case it sounds like it would also place an updated README.md in the package source folder itself, right?

Correct. README files should be stored in the package as they can be visualized in the Github preview online.

Reply:
Okay, in that case, I agree with keeping the template as optional.